src/resources/inspections.py

```diff
@@ -0,0 +1,12 @@
+from flask.json import jsonify
+from flask_restful import Resource
+
+from repositories import InspectionRepository
+from transformers import InspectionTransformer
+
+
+class InspectionsResource(Resource):
+    @staticmethod
+    def get():
+        inspections = InspectionRepository.all()
+        return jsonify({"data": [InspectionTransformer.transform(inspection) for inspection in inspections]})
```

Comment:
tutaj nie trzeba dawać tego jsonify() kiedy flask_restful jest importowany
 data -> inspections? choć nie jestem pewien :) 
`{"inspections": [InspectionTransformer.transform(inspection) for inspection in inspections]}`